packages/plugin-ext/src/plugin/in-plugin-filesystem-proxy.ts

```diff
@@ -48,6 +71,27 @@ export class InPluginFileSystemProxy implements theia.FileSystem {
             throw this.handleError(error);
         }
     }
+    async delete(uri: Uri, options?: { recursive?: boolean, useTrash?: boolean }): Promise<void> {
+        try {
+            await this.proxy.$delete(uri, { ...{ recursive: false }, ...options });
```

Comment:
[quality]: strange that you sometimes `return` with the `Promise<void>`, sometimes you don't. Why do you `await`, please explain?

Reply:
removed all awaits and leave only return now